Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2364,16 +2367,16 @@ namespace OpenInfraPlatform {
 					// Get Clothoid Constant
 					double A = clothoid->ClothoidConstant * this->UnitConvert()->getLengthInMeterFactor();
 					// Interpret polinomial constant for the following integral computations
-					std::vector<double> polynomialConstants = { 0., 0., A*sqrt(2) / 2, 0., 0. };
+					std::vector<double> polynomialConstants = { 0., 0., sqrt(2) /A*A* 2, 0., 0. };//incorrect? 

```

Comment:
be careful with `(...)` - what currently stands here is `(((sqrt(2) / A) *A) *2`

Reply:
note, this is changed in the latest sources of Enrich => spiral.cpp / spiral.h.

it was a different interpretation of the n-th order polynomial spirals (or better said change in curvature (
[Uploading IFCRail_Spirals-20210714.pptx…])

with the most recent interpretation (also as delivered to bSI) the conversion is very simple, i.e. A (as ClothoidConstant) is the linear component in the polynomial. This means practically any Clothoid can be represented in a Second, Third or SeventhOrderPolynomailSpiral, however the other way around, an IfcClothoid definition does not contain a constant term and therefore cannot represent any generic linear polynomial without using the Position. Note that the Position is ignored / irrelevant in case of use in a segment